applications/asset_tracker_v2/tests/cloud_codec_nrf_mocked_cjson/src/nrf_cloud_codec_mocked_test.c

```diff
@@ -0,0 +1,576 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <unity.h>
+#include <stdbool.h>
+
+#include "cloud_codec.h"
+#include <cJSON_os.h>
+#include <errno.h>
+
+#include "mock_json_helpers.h"
+#include "mock_date_time.h"
+#include "mock_cJSON_to_mock.h"
+#include "float.h"
+
+
+struct cloud_codec_data codec;
+int ret;
+
+void setUp(void)
+{
+	__wrap_cJSON_Init_Expect();
```

Comment:
Since it is a setup function, technically there is no need to 'expect' stuff here. You can use 'Ignore' instead. If you want to expect the CJSON_Init, then create a separate test for it. I dont have a strong opinion though.

Reply:
Okay, setting this to ignore.